packages/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerDetectorView.kt

```diff
@@ -102,24 +103,20 @@ class RNGestureHandlerDetectorView(context: Context) : ReactViewGroup(context) {
     val registry = RNGestureHandlerModule.registries[moduleId]
       ?: throw Exception("Tried to access a non-existent registry")
 
-    for (tag in nativeHandlersToAttach) {
+    for (tag in attachedNativeHandlers) {
```

Comment:
Maybe a cosmetic note, but I don't like that we iterate through `attachedNativeHandlers` and call `attachHandlerToView` - it looks like we try to attach it again, which is not the case.